pom.xml

```diff
@@ -112,7 +112,7 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <projectId>${googleCloudProjectId}</projectId>
+          <projectId>spring21-sps-2</projectId>
```

Comment:
I think this variable is already set on line 19. This doesn't change anything really as long as we don't change the one on line 19 and they go out of sync.